app/controllers/api_controller/accounts.rb

```diff
@@ -4,7 +4,7 @@ module Accounts
     # Accounts Subcollection Supporting Methods
     #
     def accounts_query_resource(object)
```

Comment:
Needed today for formal subcollections.  i.e. GET /api/vms/:id/accounts,  once we implement virtual subcollections (based on virtual attrs/reflections) it won't be needed.